src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1309,6 +1310,14 @@ window.qBittorrent.DynamicTable = (function() {
                 td.set('text', value);
                 td.set('title', value);
             };
+
+            // reannounce
+            this.columns['reannounce'].updateTd = function(td, row) {
+                const value = this.getRowValue(row);
+                const time = (value > 0) ? window.qBittorrent.Misc.friendlyDuration(value) : '';
```

Comment:
There is no `Hide zero and infinity values` option in WebUI. Isn't it inconsistent with everything else?
I have no problem with it either way.

Reply:
> if it makes sense as "infinitely delayed" reannounce.

Any value is meaningless in case of stopped torrent, IMO. `∞` could be acceptable in this case if you prefer to display something instead of nothing.
But it can be 0 also in case when timeout is just expired but torrent isn't announced yet. In this case, `∞` is hardly appropriate.